Doc/whatsnew/3.12.rst

```diff
@@ -958,9 +958,9 @@ Porting to Python 3.12
   supported, but does not fully support multiple inheritance
   (:gh:`95589`), and performance may be worse.
   Classes declaring :const:`Py_TPFLAGS_MANAGED_DICT` should call
-  :c:func:`_PyObject_VisitManagedDict` and :c:func:`_PyObject_ClearManagedDict`
+  :c:func:`!_PyObject_VisitManagedDict` and :c:func:`!_PyObject_ClearManagedDict`
   to traverse and clear their instance's dictionaries.
-  To clear weakrefs, call :c:func:`PyObject_ClearWeakRefs`, as before.
+  To clear weakrefs, call :c:func:`!PyObject_ClearWeakRefs`, as before.
```

Comment:
As this function is part of not only the public API but the Limited API, isn't this a legitimate warning as it should be documented, not ignored?

```suggestion
  To clear weakrefs, call :c:func:`PyObject_ClearWeakRefs`, as before.
```

Reply:
I wasn't sure. Looking again, I see it listed at https://docs.python.org/3/c-api/stable.html#contents-of-limited-api without a link (and so currently undocumented).

What do you think?

Reply:
Seems like anything that's part of the limited API at least is worth documenting, no? Unless it's deprecated and only included so that existing code works, but I presume if that's the case the 3.12 What's New wouldn't be recommending it.

Reply:
Added in [`1465769` (#102513)](https://github.com/python/cpython/pull/102513/commits/1465769be3d869b2be2770ab9b81c85aafc40872), text based on this comment:

https://github.com/python/cpython/blob/bb396eece44036a71427e7766fbb8e0247373102/Objects/weakrefobject.c#L934-L941